Controller/GalleryAdminController.php

```diff
@@ -66,4 +62,27 @@ public function listAction(Request $request = null)
             'csrf_token' => $this->getCsrfToken('sonata.batch'),
         ));
     }
+
+    /**
+     * Sets the admin form theme to form view. Used for compatibility between Symfony versions.
+     *
+     * @param FormView $formView
+     * @param string   $theme
+     */
+    private function setFormTheme(FormView $formView, $theme)
+    {
+        $twig = $this->get('twig');
+
+        try {
+            $twig
+                ->getRuntime('Symfony\Bridge\Twig\Form\TwigRenderer')
+                ->setTheme($formView, $theme);
+        } catch (\Twig_Error_Runtime $e) {
```

Comment:
Maybe we can use this controller as an example of how to change formTheme. You are right!